main/EventSystem.cpp

```diff
@@ -3670,6 +3669,14 @@ std::string CEventSystem::nValueToWording(const unsigned char dType, const unsig
 	else if (lstatus == "")
 	{
 		lstatus = sValue;
+		//OJO if lstatus  is still empty we use nValue for lstatus. ss for conversion
+		std::stringstream ss;
+        	lstatus = sValue;
```

Comment:
It looks lstatus is assigned the value of sValue twice (as done above your comment)